### PR TITLE
Feature/increase otp request timeouts

### PR DESCRIPTION
**Pull request by flibbertigibbet**
Pending azavea/ansible-opentripplanner/pull/13.

Large isochrones may hit the 30s timeout; this increases the timeout to 90s for OTP/gunicorn/nginx, and tweaks the isochrone precision for some requests, to speed processing.

### CODE REVIEW

File: deployment/ansible/group_vars/all [hunk 1]
```diff
@@ -6,6 +6,7 @@ otp_process_mem: 2G
 otp_repo: "https://github.com/OpenTripPlanner/OpenTripPlanner.git"
 otp_version: "otp-0.14.0"
 otp_jarfile: "./target/otp-0.14.0.jar"
+otp_session_timeout: 90
```
Comment (kshepard) [inline deployment/ansible/group_vars/all]: Maybe add an "_seconds" to this variable name to clearly indicate the unit.

**Comment by kshepard** [comment]
+1

**Comment by notthatbreezy** [comment]
One thing we'll have to watch out for when this gets deployed is making sure that the default timeouts for the ELBs are incremented, otherwise they'll still close their connections after 60s

**Comment by flibbertigibbet** [comment]
Good to know.

**Pull request merged**